src/utils/mavenUtils.ts

```diff
@@ -64,4 +66,11 @@ export namespace mavenUtils {
         }
         return result.cmdOutput;
     }
+
+    function replaceParameterQuotationMark(input: string): string {
```

Comment:
I would rather create a function that formats the arg _before_ you pass it in rather than try to change it afterwards. If you format it correctly from the start, you don't have to worry about all the weird edge cases. @jdneo mentioned one edge case, but here's another: JavaProjectCreator uses the quotes like this `-DgroupId="${groupId}"` instead of this `"-DgroupId=${groupId}"`

I suggest you create a function like this:
```typescript
function formatMavenArg(key: string, value: string): string {
    return `-${key}=${quotationMark}${value}${quotationMark}`
}
```

which you leverage in [JavaProjectCreator](https://github.com/Microsoft/vscode-azurefunctions/blob/master/src/commands/createNewProject/JavaProjectCreator.ts) and [JavaFunctionCreator](https://github.com/Microsoft/vscode-azurefunctions/blob/master/src/commands/createFunction/JavaFunctionCreator.ts)